tools/docker/Dockerfile.jinja

```diff
@@ -111,6 +114,14 @@ WORKDIR /root/schbench
 RUN make \
     && cp schbench /usr/local/benchmark/
 
+# Build memaslap for memcached
+WORKDIR /root/libmemcached-1.0.18
+RUN LDFLAGS='-lpthread' CPPFLAGS='-fcommon -fpermissive' CFLAGS='-fpermissive -fcommon' \
+    ./configure --enable-memaslap --prefix=/usr/local/benchmark/libmemcached \ 
+    && CPPFLAGS='-fcommon' make -j \ 
+    && make install \
+    && ldconfig
```

Comment:
Is the ending `ldconfig` really necessary?

Reply:
Oh I think it's not really necessary. Thanks! I will delete it.